dbt/include/bigquery/macros/catalog.sql

```diff
@@ -7,30 +7,42 @@
   {%- else -%}
 
   {%- set query -%}
-    with tables as (
+    with materialized_views as (
         select
-            project_id as table_database,
-            dataset_id as table_schema,
-            table_id as original_table_name,
+            table_catalog as project_id,
+            table_schema as dataset_id,
+            table_name as table_id
+        from {{ information_schema.replace(information_schema_view='MATERIALIZED_VIEWS') }}
+    ),
+    tables as (
+        select
+            tables.project_id as table_database,
+            tables.dataset_id as table_schema,
+            tables.table_id as original_table_name,
 
-            concat(project_id, '.', dataset_id, '.', table_id) as relation_id,
+            concat(tables.project_id, '.', tables.dataset_id, '.', tables.table_id) as relation_id,
 
-            row_count,
-            size_bytes as size_bytes,
+            tables.row_count,
+            tables.size_bytes as size_bytes,
             case
-                when type = 1 then 'table'
-                when type = 2 then 'view'
+                when materialized_views.table_id is not null then 'materialized view'
```

Comment:
given that the config block, should this be `materialized_view` instead of here with the space. A nit only. Just thinking about consistency of output.

Reply:
I thought about this in general for the three related PRs. It looks like we always return it with a space, but we return all caps for some adapters, all lowercase for others, and sometimes we return `"BASE TABLE"` whereas others return `"table"`. I feel like all of these should be standardized (probably to the `RelationType` enum), but I don't know if Cloud is depending on this list of values somehow. Given all of that, `'materialized view'` seemed the most consistent for now.

Reply:
you've clearly weighed the options which is what I was wondering most. I agree standardization is kind of a separate, high-level thing we'll need to address some day if the need presents itself. For now, I can live with the current state of affairs (even if from the outside I'd be asking 'wat')